ts/util/emoji.ts

```diff
@@ -33,17 +36,18 @@ export function getEmojiSizeClass(str: string): SizeClassType {
   const emojiCount = getCountOfAllMatches(str);
   if (emojiCount > 6) {
     return 'small';
-  } else if (emojiCount > 4) {
+  }
+  if (emojiCount > 4) {
```

Comment:
What benefit of flattening this to just if statements?

Reply:
well it's just that there is a return on the previous if, so no point of having a else if .
this is a default setting of eslint (probably part of airbnb-base)

Reply:
Cool